src/hazelcore/Action_Exp.re

```diff
@@ -17,30 +17,6 @@ let operator_of_shape =
   | SVBar => None
   };
 
-let shape_of_operator =
```

Comment:
leave this one in -- it's a check that new operators have a corresponding shape (leave a comment to this effect)